report/src/problem_analysis/presence/environmental.tex

```diff
@@ -0,0 +1,13 @@
+\subsection{Presence detection using $\text{CO}_{2}$ concentration}\label{sec:presence_env_co2}
+As illustrated in the example above, it is possible to use $\text{CO}_{2}$ levels to detect presence in a room.
+This approach uses $\text{CO}_{2}$ sensors to detect the presence of a person in a room, and is based on the assumption that the presence of a person will increase the $\text{CO}_{2}$ concentration in the room.\cite{gruberCO2SensorsOccupancy2014}
+
+However, there are several causes of concern with this approach. The most important ones are:
+\begin{itemize}
+    \item Ventilation flow rate matters as it affects the $\text{CO}_{2}$ concentration in the room. However, in some areas, this could be outside of the control of the user.
+    \item Similarly, opening a door or window will also affect the $\text{CO}_{2}$ concentration in the room.
+    \item There is a time delay between the presence of a person and a noticeable increase in $\text{CO}_{2}$ concentration. The more people that are present, the shorter the time delay will be.
+\end{itemize}
+\cite{gruberCO2SensorsOccupancy2014,longoAccurateOccupancyEstimation2019}
+
+Various studies have been conducted to determine the accuracy of $\text{CO}_{2}$ sensors for occupancy detection. \citeauthor{longoAccurateOccupancyEstimation2019}\cite{longoAccurateOccupancyEstimation2019} summarizes some of these and show the accuracy of $\text{CO}_{2}$ measurements for occupancy detection is between 50\% and 99\%, depending on task and amount of occupants to estimate.
```

Comment:
```suggestion
Various studies have been conducted to determine the accuracy of $\text{CO}_{2}$ sensors for occupancy detection. 
\citeauthor{longoAccurateOccupancyEstimation2019}\cite{longoAccurateOccupancyEstimation2019} summarizes some of these and show the accuracy of $\text{CO}_{2}$ measurements for occupancy detection is between 50\% and 99\%, depending on task and amount of occupants to estimate.
```

Reply:
It would be nice to have a single sentence saying when it works well and when it does not.

Reply:
´´´
Therefore, the environmental prerequisite for this approach is that the ventilation flow rate is rather constant, and the room is not exposed to a lot of outside air during measurements. This may be difficult to achieve in some environments, such as offices, where the ventilation flow rate is often not constant, and the room is exposed to a lot of outside air, given open windows or doors.
´´´

Thoughts?